tests/base.py

```diff
@@ -109,43 +118,45 @@ def expected_metadata(self):
         return {
             'accounts': {
                 self.PRIMARY_KEYS: {'id'},
-                self.REPLICATION_METHOD: 'INCREMENTAL',
+                self.REPLICATION_METHOD: self.INCREMENTAL,
                 self.REPLICATION_KEYS: {'last_modified_time'}
```

Comment:
Add `OBEY_START_DATE` and use it in the start_date test.

Reply:
Added `OBEY_START_DATE` for the `start_date` test.